app/templates/events/view/tickets/discount-codes.hbs

```diff
@@ -0,0 +1,93 @@
+<div class="ui stackable grid">
+  <div class="row">
+    <div class="eight wide column">
+      <h2 class="ui header">
+        {{t 'Discount Codes'}}
+      </h2>
+    </div>
+    <div class="eight wide {{unless device.isMobile 'right aligned'}} column">
+      <button class="ui teal button">
+        <i class="plus icon"></i>
+        {{t 'Create a discount'}}
+      </button>
+    </div>
+  </div>
+  <div class="row">
+    <div class="ui hidden divider"></div>
+    <div class="ui {{if device.isMobile 'vertical'}} basic buttons">
+      <div class="ui button active">
+        {{t 'All'}}
+      </div>
+      <div class="ui button">
+        {{t 'Active'}}
+      </div>
+      <div class="ui button">
+        {{t 'Inactive'}}
+      </div>
+    </div>
+    <table class="ui very basic table">
```

Comment:
@anu-007 @niranjan94 I dont think that will help here. You can either add a `compact` class here of you can specify the width of each column like this
```
<table class="ui very basic table">
  <thead>
    <tr>
      <th class="ten wide">Head 1</th>
      <th class="six wide">Head 2</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td>one</td>
      <td>two</td>
    </tr>
  </tbody>
</table>
```